src/components/app/details/appDetails/AppDetails.tsx

```diff
@@ -597,9 +511,7 @@ export const Details: React.FC<DetailsType> = ({
             if (!response.result.isPipelineTriggered && response.result.releaseMode === ReleaseMode.NEW_DEPLOYMENT) {
                 setResourceTreeFetchTimeOut(false)
                 setLoadingResourceTree(false)
-                setAppDetails(null)
```

Comment:
Why are we removing this. in this case it should not attempt to fetch the resource tree as the app is not yet deployed